### ISSUE TITLE
Slow loading time on iOS with NativeScript 4.0.0

### ISSUE THREAD

**Issue by NickIliev** [manijak/nativescript-photoviewer#24]
I am using [this basic test project](https://github.com/NickIliev/NS-Issues-2018/tree/master/others/t_1163464) and on iOS emulator and device the loading time for a gallery with 4 images is approximately 10-15 seconds and during this time the UI is completely blocked.

Steps to reproduce:
- use this app
- tap the button to l[oad the images](https://github.com/NickIliev/NS-Issues-2018/blob/master/others/t_1163464/app/main-page.ts#L20)

Tested on iPhone X with iOS 11.3

**Comment by AradhanaR** [comment]
I am facing the same issue, any fix known?

**Comment by manijak** [comment]
@chrisbekas you can take a look at the demo apps I've added. It works fine on vanilla tns apps, but on angular the gui and spinner still hang. have not figured out why that is.

**Comment by mircobabini** [comment]
Since this is not fixed anymore and the repo seems unmaintained, can you please suggest a valid alternative?

**Comment by manijak** [comment]
@mircobabini Yeah I don't think there will be any new updates or features in this repo. 

My recommendation would be to combine following plugins to create a gallery:
* nativescript-carousel: https://github.com/manijak/nativescript-carousel
* nativescript-image-zoom (or any other image plugin of your choice)
* Modal view in you app

This way you will have complete control of the components and will be able to create similar, if not better, looking gallery. I did this on Android for my app, and it works great.
